spring-integration-gcp/pom.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		 xmlns="http://maven.apache.org/POM/4.0.0"
+		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<artifactId>spring-cloud-gcp</artifactId>
+		<groupId>org.springframework.cloud</groupId>
+		<version>1.0.0.BUILD-SNAPSHOT</version>
+	</parent>
+	<name>Spring Integration GCP</name>
+	<description>Integration modules for Google Cloud Platform</description>
+	<artifactId>spring-integration-gcp</artifactId>
+	<organization>
```

Comment:
Not sure that this will go to SI Core once. We can't depend on Spring Cloud there, since Spring Cloud depends on SI Core... 
And not sure that it will go its separate project. That is fully fine with me as a module in this `spring.cloud` member.
For example Spring Batch has some SI module as well: https://github.com/spring-projects/spring-batch/tree/master/spring-batch-integration, but that isn't `org.springframework.integration` artifact any way.

Also any third party community may decide to implement they own channel adapters for some peculiar protocol (MQTT-NS ?) and they are really fully free to build their own group and package structure...